engine/flink/avro-util/src/main/java/org/apache/flink/formats/avro/typeutils/LogicalTypesAvroTypeInfo.java

```diff
@@ -44,60 +33,20 @@
  *
  * <p>This class is checked by the AvroPojoTest.
  */
-public class LogicalTypesAvroTypeInfo<T extends SpecificRecordBase> extends PojoTypeInfo<T> {
+public class LogicalTypesAvroTypeInfo<T extends SpecificRecordBase> extends AvroTypeInfo<T> {
```

Comment:
+1 for catching up that now it can be simpler

Reply:
only because it didn't compile :)